README.md

```diff
@@ -12,25 +12,28 @@
   <a href="https://coveralls.io/github/Microsoft/BotFramework-WebChat?branch=master"><img src="https://coveralls.io/repos/github/Microsoft/BotFramework-WebChat/badge.svg?branch=master" alt="Coverage Status" /></a>
 </p>
 
-# Migrating from v3 to v4 of Web Chat
+## Migrating from Web Chat v3 to v4
 
-There are three possible paths that migration might take when migrating from v3 to v4. First please compare your beginning scenario:
+There are three possible paths that migration might take when migrating from v3 to v4. First, please compare your beginning scenario:
 
-### My current site uses an `iframe` obtained from Azure Bot Services (which uses v3 of Web Chat). I want to upgrade to v4.
-- At this time, it is not possible to upgrade to v4 of Web Chat by pasting an `iframe` into your site. Please follow the implementation of sample [`01.a.getting-started-full-bundle`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/01.a.getting-started-full-bundle) to upgrade to Web Chat v4.
+### My current website integrate Web Chat using an `<iframe>` element obtained from Azure Bot Services. I want to upgrade to v4.
 
-### My Bot's webpage is rendering Web Chat v3 and uses customization options provided by Web Chat, no customization at all, or very little of my own customization that was not available with Web Chat.
-- Please follow the implementation of sample [`01.c.getting-started-migration`](https://github.com/Microsoft/BotFramework-WebChat/tree/master/samples/01.c.getting-started-migration) to convert your webpage from v3 to v4 of Web Chat.
+Starting from May 2019, we are rolling out v4 to websites that integrate Web Chat using `<iframe>` element. Please refer to [this article](https://github.com/Microsoft/BotFramework-WebChat/treemaster/packages/embed/README.md) for information on integrating Web Chat using `<iframe>`.
```

Comment:
```suggestion
Starting from May 2019, we are rolling out v4 to websites that integrate Web Chat using `<iframe>` element. Please refer to [the embed documentation](https://github.com/Microsoft/BotFramework-WebChat/treemaster/packages/embed/README.md) for information on integrating Web Chat using `<iframe>`.
```

Don't forget that saying 'this documentation' is not good accessibility, since it doesn't tell the user what they will be clicking.